gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -118,3 +121,20 @@ def get_python_type_from_bigquery_type(bigquery_type):
   if bigquery_type not in _BIG_QUERY_TYPE_TO_PYTHON_TYPE_MAP:
     raise ValueError('Invalid BigQuery type: %s' % bigquery_type)
   return _BIG_QUERY_TYPE_TO_PYTHON_TYPE_MAP[bigquery_type]
+
+
+def get_vcf_type_from_bigquery_type(bigquery_type):
+  # type: (str) -> str
+  """Returns VCF type based on BigQuery type."""
+  if bigquery_type not in _BIG_QUERY_TYPE_TO_VCF_TYPE_MAP:
+    raise ValueError('Invalid BigQuery type: %s' % bigquery_type)
```

Comment:
usually errors are lowercase

Reply:
@pgrosu while that may be true in some other projects, in our project we don't have this convention and we follow Beam's conventions.

Reply:
@arostamianfar That is a fair point :)